src/universe/index.ts

```diff
@@ -254,6 +255,81 @@ export interface UniverseAnalyticsEventsOptions {
   end: string
   datepart?: string
 }
+export interface UniverseMe {
+  (skipInterceptors?: boolean): Promise<MeData | never>
+  addTemplateFavorite: (favorite: StaffMessageTemplateFavorite) => Promise<staff.StaffRawPayload | never>
+  removeTemplateFavorite: (id: string) => Promise<staff.StaffRawPayload | never>
+}
+
+const me: UniverseMe = async function (this: Universe, skipInterceptors = false): Promise<MeData | never> {
```

Comment:
question is tehre maybe a better place for me rather than in src/universe/index.ts? The file is kinda large...